docs/reference/ml/trained-models/apis/update-trained-model-deployment.asciidoc

```diff
@@ -37,17 +37,34 @@ include::{es-ref-dir}/ml/ml-shared.asciidoc[tag=deployment-id]
 [[update-trained-model-deployment-request-body]]
 == {api-request-body-title}
 
+`adaptive_allocations`::
```

Comment:
I would also refer to `adaptive_allocations` in the description section in the same paragraph as `number_of_allocations`. The current
> You can either increase or decrease the number of allocations of such a deployment.
is accurately anymore, because you can also change `adaptive_allocations`.